app/upgrades/v26/upgrades.go

```diff
@@ -15,14 +36,33 @@ func CreateUpgradeHandler(
 	bpm upgrades.BaseAppParamManager,
 	keepers *keepers.AppKeepers,
 ) upgradetypes.UpgradeHandler {
-	return func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+	return func(context context.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		ctx := sdk.UnwrapSDKContext(context)
 		// Run migrations before applying any other state changes.
 		// NOTE: DO NOT PUT ANY STATE CHANGES BEFORE RunMigrations().
 		migrations, err := mm.RunMigrations(ctx, configurator, fromVM)
 		if err != nil {
 			return nil, err
 		}
 
+		// UNFORKING v2 TODO: I think there is just one new gov param that is not registered, which is why this is needed. Need to figure out what it is rather than re-setting all params.
```

Comment:
.

Reply:
nit: can we do something like:
 ```
 params := GetTheParams()
 params.TheNewOne := value
 keepers.GovKeeper.Params.Set(ctx, params)
 ```
 instead of hardcoding all those values?

Reply:
Yeah, I just need to figure out what the one value is 😅